base_exception/i18n/am.po

```diff
@@ -0,0 +1,227 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# * sale_exception
+# 
+# Translators:
+msgid ""
+msgstr ""
+"Project-Id-Version: sale-workflow (9.0)\n"
```

Comment:
Maybe could be better to generate new po files, and update this lines related with module sale_exception on project sale-workflow instead of base_exception on server-tools.